core/src/main/java/tech/tablesaw/analytic/AggregateFunctions.java

```diff
@@ -213,41 +233,188 @@ public Double getValue() {
   }
 
   static class Min<T extends Number> extends WindowDependentAggregateFunction<T> {
-
     @Override
-    AggregateFunction<T, Double> fixedFunction() {
-      throw new UnsupportedOperationException("Analytic Function Min Is Not implemented");
+    AppendAggregateFunction<T, Double> functionForAppendWindows() {
+      return new AppendAggregateFunction<T, Double>() {
+        private Double min = Double.NaN;
+
+        @Override
+        public void addRightMost(T newValue) {
+          if (Double.isNaN(min)) {
+            min = newValue.doubleValue();
```

Comment:
It's an infrequent case, so it doesn't necessarily need to be dealt with here, but any comparable type could support min.

Reply:
Yes. That should be an easy fix.